models/rust/src/3.0/files/system_pricing_plans.rs

```diff
@@ -0,0 +1,79 @@
+#[cfg(feature = "pyo3")]
+use pyo3::prelude::*;
+
+#[cfg(feature = "napi")]
+use napi_derive::napi;
+
+use serde::{Deserialize, Serialize};
+
+use crate::v3_0::types::*;
+
+file_struct!(SystemPricingPlansFile, SystemPricingPlansData);
+
+#[cfg_attr(feature = "napi", napi(object))]
+#[cfg_attr(feature = "pyo3", pyclass(get_all, set_all))]
+#[serde_with::skip_serializing_none]
+#[derive(Serialize, Deserialize, Debug, Clone)]
+/// Contains one object per vehicle that is currently deployed in the field.
+pub struct SystemPricingPlansData {
+    pub plans: Vec<SystemPricingPlan>,
+}
+
+#[cfg_attr(feature = "napi", napi(object))]
+#[cfg_attr(feature = "pyo3", pyclass(get_all, set_all))]
+#[serde_with::skip_serializing_none]
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct SystemPricingPlan {
+    /// Identifier for a pricing plan in the system.
+    pub plan_id: PricingPlanID,
+    /// URL where the customer can learn more about this pricing plan.
+    pub url: Option<URL>,
+    /// Name of this pricing plan.
+    pub name: Option<Vec<LocalizedString>>,
+    /// Currency used to pay the fare.
+    pub currency: Currency,
+    /// Fare price, in the unit specified by currency.
+    ///
+    /// In case of non-rate price, this field is the total price.
+    /// In case of rate price, this field is the base price that is charged only once per trip (typically the price for unlocking) in addition to per_km_pricing and/or per_min_pricing.
+    pub price: Option<NonNegativeFloat>,
+    /// The cost, described as per minute rate, to reserve the vehicle prior to beginning a rental.
+    /// This amount is charged for each minute of the vehicle reservation until the rental is initiated, or until the number of minutes defined in vehicle_types.json#default_reserve_time elapses, whichever comes first.
+    /// This field MUST NOT be combined in a single pricing plan with reservation_price_flat_rate.
+    pub reservation_price_per_min: Option<NonNegativeFloat>,
+    /// The cost, described as a flat rate, to reserve the vehicle prior to beginning a rental.
+    /// This amount is charged once to reserve the vehicle for the duration of the time defined by vehicle_types.json#default_reserve_time.
+    /// This field MUST NOT be combined in a single pricing plan with reservation_price_per_min.
+    pub reservation_price_flat_rate: Option<NonNegativeFloat>,
```

Comment:
These 2 fields were removed from v3.0 and added to v3.1-RC ([Release Notes](https://github.com/MobilityData/gbfs/releases/tag/v3.1-RC)).

```suggestion
```